sbc_translation/models/correspondence.py

```diff
@@ -23,6 +23,7 @@ class Correspondence(models.Model):
         """
     _inherit = "correspondence"
 
+    name = fields.Char(store=True)
```

Comment:
this change can be done directly in the super module (sbc_compassion), and you should make sure that the compute method is correctly decorated with api.depends

Reply:
Making the modification in the parent module makes the field uselessly stored when sbc_translation isn't installed